### DISCUSSION TITLE
How to ignore changes to files in a directory and stop them from reloading the HTTP server (Adonis V5)?

### DISCUSSION THREAD

**Discussion by bgord** [adonisjs/core#2082] [Help]
I have the following setup:

```
|-- frontend
    |-- index.tsx
    |-- util.ts
    |-- ...
|-- tsconfig.json
|-- <regular Adonis V5 files>
```

All changes to a `.ts` or `.tsx` file inside the `frontend/` directory trigger HTTP server reloading. How can I exclude this directory to prevent unnecessary reloads?

Adding `frontend` to the `exclude` section in `tsconfig.json` works, but one of the TS libraries stops me from going this way. Are there any other options?

**Comment by thetutlage** [comment]
Unfortunately not. We rely on the Typescript to provide us the set of files to watch and ignore. Is it not possible for you to have separate `tsconfig` files for your frontend and the backend?
